src/dlls/mscorrc/mscorrc.rc

```diff
@@ -1252,6 +1252,8 @@ BEGIN
     IDS_CLASSLOAD_MI_VIRTUALMISMATCH        "Method '%3' on type '%1' from assembly '%2' tried to implement a method declaration with a different virtual state."
     IDS_CLASSLOAD_MI_MUSTBEVIRTUAL          "Method '%3' on type '%1' from assembly '%2' must be virtual to implement a method on an interface or super type."
     IDS_CLASSLOAD_MI_BAD_SIG                "Type '%1' from assembly '%2' contains an invalid method implementation signature."
+    IDS_CLASSLOAD_MI_FINAL_IMPL             "Method implementation on an interface must be a final method.  Type: '%1'.  Assembly: '%2'."
```

Comment:
>Method implementation on an interface must be a final method.  Type: '%1'.  Assembly: '%2'." [](start = 45, length = 92)

Change the text to be:

"Method implementation on interface '%1' from assembly '%2' must be a final method."

To follow the pattern